src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -14,6 +14,13 @@ function branch_needs_push {
 
 
 # Checkout a branch
+function checkout_branch {
+  local branch_name=$1
+  run_command "git checkout $branch_name"
+}
```

Comment:
Ah, I was wondering about this. This is inconsistent with our naming strategy, and this file also contains a method `checkout_branch_silently`. I'm removing my changes here, and will also move the checkout_silently method over.